src/test_utils/mongo/mod.rs

```diff
@@ -523,33 +274,118 @@ impl<'a> MongoFuzzer<'a> {
+                            v: CHAIN_ID.saturating_mul(U256::from(2)).saturating_add(U256::from(35)),
+                            y_parity: Default::default(),
+                        }),
+                        chain_id: Some(U64::from(1)),
+                        blob_versioned_hashes: Default::default(),
+                        transaction_type: Some(U64::from(Into::<u8>::into(TxType::Legacy))),
+                        ..Default::default()
+                    },
+                },
+                TxType::Eip2930 => StoredTransaction {
+                    tx: reth_rpc_types::Transaction {
+                        hash: *EIP2930_TX_HASH,
+                        block_hash: Some(*BLOCK_HASH),
+                        block_number: Some(U256::from(BLOCK_NUMBER)),
+                        transaction_index: Some(U256::ZERO),
+                        from: *RECOVERED_EIP2930_TX_ADDRESS,
+                        to: Some(Address::ZERO),
+                        gas_price: Some(U128::from(10)),
+                        gas: U256::from(100),
+                        signature: Some(reth_rpc_types::Signature {
+                            r: *TEST_SIG_R,
+                            s: *TEST_SIG_S,
+                            v: *TEST_SIG_V,
+                            y_parity: Some(reth_rpc_types::Parity(true)),
+                        }),
+                        chain_id: Some(U64::from(1)),
+                        access_list: Some(Default::default()),
+                        transaction_type: Some(U64::from(Into::<u8>::into(TxType::Eip2930))),
+                        ..Default::default()
+                    },
+                },
+                _ => unimplemented!(),
+            });
+        }
+
+        Ok(StoredTransaction::arbitrary(&mut arbitrary::Unstructured::new(&{
+            (0..rnd_bytes_size).map(|_| rand::random::<u8>()).collect::<Vec<_>>()
+        }))?)
+    }
+}
+
 #[cfg(test)]
 mod tests {
     use super::*;
     use crate::eth_provider::database::types::{
         header::StoredHeader, receipt::StoredTransactionReceipt, transaction::StoredTransaction,
     };
-    use rand::Rng;
-
-    #[tokio::test]
-    async fn test_mongo_connection() {
-        // Create a mock database.
-        let database = mock_database().await;
-
-        // Retrieve a single document from the "headers" collection.
-        let _ = database.get_one::<StoredHeader>("headers", None, None).await.unwrap();
-
-        // Drop the inner MongoDB database.
-        database.inner().drop(None).await.unwrap();
-    }
 
     #[tokio::test]
     async fn test_mongo_fuzzer() {
-        let mut bytes = [0u8; 1024];
-        rand::thread_rng().fill(bytes.as_mut_slice());
-
         // Mocks a database with 100 headers and 100 transactions.
-        let database = MongoFuzzer::mock_database(&mut arbitrary::Unstructured::new(&bytes), 100, 100).await;
+        let database = MongoFuzzer::mock_database(100024, 100).await;
```

Comment:
can we maybe make a constant for `100024`?

Reply:
Yep good idea, done.